pkg/descheduler/framework/plugins/loadaware/utilization_util.go

```diff
@@ -665,14 +666,24 @@ func calcAverageResourceUsagePercent(nodeUsages map[string]*NodeUsage) (Resource
 	}
 	return average, prodAverage
 }
-func sortPodsOnOneOverloadedNode(srcNode NodeInfo, removablePods []*corev1.Pod, resourceWeights map[corev1.ResourceName]int64) {
+func sortPodsOnOneOverloadedNode(srcNode NodeInfo, removablePods []*corev1.Pod, resourceWeights map[corev1.ResourceName]int64, prod bool) {
 	weights := make(map[corev1.ResourceName]int64)
 	// get the overused resource of this node, and the weights of appropriately using resources will be zero.
-	overusedResources, _ := isNodeOverutilized(srcNode.usage, srcNode.thresholds.highResourceThreshold)
-	for or := range overusedResources {
+	var overusedResources corev1.ResourceList
+	if prod {
+		overusedResources, _ = isNodeOverutilized(srcNode.prodUsage, srcNode.thresholds.prodHighResourceThreshold)
+	} else {
+		overusedResources, _ = isNodeOverutilized(srcNode.usage, srcNode.thresholds.highResourceThreshold)
+	}
+	resourcesThatExceedThresholds := map[corev1.ResourceName]resource.Quantity{}
+	for or, used := range overusedResources {
+		usedCopy := used.DeepCopy()
 		weights[or] = resourceWeights[or]
+		usedCopy.Sub(*srcNode.thresholds.highResourceThreshold[or])
```

Comment:
Here need to be modified as if prod{ usedCopy.Sub(*srcNode.thresholds.prodHighResourceThreshold)}

Reply:
ok